### PR TITLE
[air-output] better wording for empty config.

### PR DESCRIPTION
**Pull request by xwjiang2010**
## Why are these changes needed?
better wording for empty config.

Without this, we have
```
xxx started with configuration: 
{}
```

with this, we have
```
xxx started.
```

## Related issue number
#33822

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by krfricke — COMMENTED** [review]
Is this PR still relevant?

**Comment by xwjiang2010** [comment]
yes it is.  Current wording is a bit confusing (see description of the PR).

**Comment by krfricke** [comment]
I see, the reason I'm asking is because the diff looks unrelated (or maybe the change has been included in a different PR already?)
Screenshot 2023-03-31 at 1 50 35 PM

File: python/ray/tune/experimental/output.py [hunk 1]
```diff
@@ -716,7 +716,7 @@ def on_checkpoint(
     def on_trial_start(self, iteration: int, trials: List[Trial], trial: Trial, **info):
         if self._verbosity < self._start_end_verbosity:
             return
-        has_config = trial.config is not None
+        has_config = not (not trial.config)
```
Comment (krfricke) [inline python/ray/tune/experimental/output.py:719]:
```suggestion
        has_config = bool(trial.config)
```

**Review by krfricke — APPROVED** [review]

**Pull request merged**